cmd/make-bm-worker/templates/templates_test.go

```diff
@@ -19,7 +19,7 @@ func compareStrings(t *testing.T, s1, s2 string) bool {
 		maxLines = len(s2Lines)
 	}
 
-	for i := 0; i < maxLines; i++ {
+	for i := range maxLines {
```

Comment:
`range` over int is Go 1.22+ feature, but we require 1.22 minimum for all modules, so this is fine.

Reply:
👍